frontend/app/routes/_gcweb-app.personal-information.home-address.confirm.tsx

```diff
@@ -85,13 +91,23 @@ export default function PersonalInformationHomeAddressConfirm() {
           <div className="py-4 sm:grid sm:grid-cols-3 sm:gap-4 sm:py-6">
             <dt className="font-medium">{t('personal-information:home-address.confirm.new-home-address')}</dt>
             <dd className="mt-3 sm:col-span-2 sm:mt-0">
-              <Address
-                address={newHomeAddress.address}
-                city={newHomeAddress.city}
-                provinceState={regionList.find((region) => region.provinceTerritoryStateId === newHomeAddress.province)?.provinceTerritoryStateId}
-                postalZipCode={newHomeAddress.postalCode}
-                country={countryList.find((country) => country.countryId === newHomeAddress.country)?.[i18n.language === 'fr' ? 'nameFrench' : 'nameEnglish'] ?? ' '}
-              />
+              {useSuggestedAddress ? (
+                <Address
+                  address={suggestedAddress.address}
+                  city={suggestedAddress.city}
+                  provinceState={regionList.find((region) => region.provinceTerritoryStateId === suggestedAddress.province)?.provinceTerritoryStateId}
+                  postalZipCode={suggestedAddress.postalCode}
+                  country={countryList.find((country) => country.countryId === suggestedAddress.country)?.[i18n.language === 'fr' ? 'nameFrench' : 'nameEnglish'] ?? ' '}
+                />
+              ) : (
+                <Address
+                  address={newHomeAddress.address}
+                  city={newHomeAddress.city}
+                  provinceState={regionList.find((region) => region.provinceTerritoryStateId === newHomeAddress.province)?.provinceTerritoryStateId}
+                  postalZipCode={newHomeAddress.postalCode}
+                  country={countryList.find((country) => country.countryId === newHomeAddress.country)?.[i18n.language === 'fr' ? 'nameFrench' : 'nameEnglish'] ?? ' '}
+                />
+              )}
```

Comment:
**Nitpick/personal preference:** I don't like large, multi-line ternaries. I find them difficult to read.

Is there any better way to do this?

``` tsx
{useSuggestedAddress && <Address />}
{!useSuggestedAddress && <Address />}
```

I don't know that this is more readable. Maybe this kind of code is just a turd no matter what you do.

Reply:
Compliments of @sebastien-comeau: [TS Playground](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAhgEwQJwKYQgSTAMxDAXhgFcJUBlYgcyvSlQQEEk0MYB+GCa26B5lOggwAXDDCoA7gAkQAW1QDWEANwAoUJFgBvRIIwAaGMACWUAJ5GADiGhwANgGEQCVAF9C8FkOx51G8GgYNCoTcE8QsLAAGRNoADocEzAEAApUyPAASkIAPmDUUPB4q2QQADdk4FQAFVRkZDMQZHNyKDh6TARCAiI9ZV8QErLKsGqs9U0g0oqqinb6CMKo9mHZsdr6xqhm1oXULv8p2FBiMCgWz1PzltiEpJT064vzHIJ855b4z-Munr7vBhBt8QGcXhMAlpjKCbuYAHJwBRXGEvGAAMjRMFSJgAjAAOMDxexwMBUYhwWj-GAAchwyGpHGhYK+YERqAAYmgxgALURM2HxVkKACipPscW5ELUAHppTBifRkGoADxKIRqGBefQQAi6QEQNwa4xmcy60wWQ2amajaptDqoXXWuZ2xaWmA2Oz2ABaJiszlcjts7ScLncRp+ZpRLQRCkN0tyQA)

Reply:
Thanks and fixed [here](https://github.com/DTS-STN/canadian-dental-care-plan/compare/80533e5d722577969bcd8f923d88caab329d3bae..6d58543589c408a034c61f8a0209748698a0befc#diff-d9502a638ae09631a8fc146cdc0eced1923c745ad240c958476d3bab381c5694)